tests/aws/scenario/events_microservices/test_eventbridge_microservices.py

```diff
@@ -0,0 +1,347 @@
+import json
+import logging
+import os
+
+import aws_cdk as cdk
+import aws_cdk.aws_events as events
+import aws_cdk.aws_events_targets as targets
+import aws_cdk.aws_iam as iam
+import aws_cdk.aws_lambda as _lambda
+import aws_cdk.aws_s3 as s3
+import aws_cdk.aws_sqs as sqs
+import pytest
+
+from localstack.testing.pytest import markers
+from localstack.utils.files import load_file
+from tests.aws.services.events.helper_functions import sqs_collect_messages
+
+LOG = logging.getLogger(__name__)
+
+REGION_PRIMARY = "us-east-1"
+REGION_SECONDARY = "eu-central-1"
+
+STACK_NAME_PRIMARY = "EventsStackPrimaryRegion"
+STACK_NAME_SECONDARY = "EventsStackSecondaryRegion"
+
+S3_BUCKET_NAME = "eventbridge-secondary-s3-bucket"
+
+EVENT_BUS_NAME_PRIMARY = "event_bus_primary"
+EVENT_BUS_NAME_SECONDARY = "event_bus_secondary"
+
+SOURCE_PRODUCER_PRIMARY_ONE = "producer-primary-one"
+SOURCE_PRODUCER_PRIMARY_TWO = "producer-primary-two"
+SOURCE_PRODUCER_SECONDARY = "producer-secondary"
+
+DEFAULT_MESSAGE = {
+    "command": "update-account",
+    "payload": {"acc_id": "0a787ecb-4015", "sf_id": "baz"},
+}
+
+
+def _read_file_as_string(filename: str) -> str:
+    file_path = os.path.join(os.path.dirname(__file__), filename)
+    return load_file(file_path)
+
+
+class TestEventsCrossAccountRegionScenario:
+    @pytest.fixture(scope="class", autouse=True)
+    def infrastructure(self, infrastructure_setup, account_id):
+        infra = infrastructure_setup(
+            namespace="MultiStackDeploymentEventsCrossAccountRegionScenario", force_synth=True
+        )
+        ##########################
+        # Primary region / account
+        ##########################
+
+        stack_primary = cdk.Stack(
+            infra.cdk_app,
+            STACK_NAME_PRIMARY,
+            # cross_region_references=True,
+            env=cdk.Environment(region=REGION_PRIMARY),
```

Comment:
I would rather prefer to use the `region` and `secondary_region_name` fixtures here.

Reply:
fixed in: https://github.com/localstack/localstack/pull/10968/commits/6cdc36b96c5c0a8c15687c38ea2898711f92407d